python-sdk/src/astro/sql/operators/merge.py

```diff
@@ -73,8 +83,62 @@ def execute(self, context: Context) -> BaseTable:
             target_conflict_columns=self.target_conflict_columns,
             source_to_target_columns_map=self.columns,
         )
+        context["ti"].xcom_push(key="merge_query", value=str(db.sql))
```

Comment:
Why is this additional `str` needed?

Reply:
I think because sqlalchemy insert statement is not serializable

Reply:
Ah OK that makes sense, I thought `sql` is the raw string.

Reply:
@uranusjr we wanted to typecast this to string before pushing as per discussion: https://astronomer.slack.com/archives/C03868KGF2Q/p1665754738297099